wherehows-web/app/utils/api/datasets/acl-access.js

```diff
@@ -0,0 +1,44 @@
+// import Ember from 'ember';
+// const { $: { getJSON } } = Ember;
+
+import { getJSON, postJSON } from 'wherehows-web/utils/api/fetcher';
+
+// TODO: Defined check current user can access ACL tab and ACL users list
+
+/**
+ * Defined base URL for acl server 
+ * @type {string}
+ */
+const aclUsersList = 'http://localhost:3000/elements/';
+
+/**
+ * Defined URL for checking current user accessibility
+ * @param {string} userName the current user name
+ * @return {string} URL check current user accessibility
+ */
+
```

Comment:
You don't need spaces between the function definition and the attached jsdoc comment